Makefile

```diff
@@ -667,6 +671,10 @@ superclean: clean supercleanfiles
 		echo $(SUPERCLEAN_FILES) | tr ' ' '\n'; \
 		$(RM) $(SUPERCLEAN_FILES); \
 	fi
+	@ \
+	if [ -d $(CUB_DIR) ]; then \
```

Comment:
Yeah, I was thinking about to make it consistent with the previous case... Never mind, reverted to `@- $(RM)...`.